src/FOMObot/Helpers/Bot.hs

```diff
@@ -56,3 +63,21 @@ alertFOMOChannel channelID = do
     Slack.sendMessage fomoChannel message
   where
     message = "Check out <#" <> (channelID ^. Slack.getId) <> ">"
+
+processCommand :: Slack.Event -> Bot ()
+processCommand (Slack.Message cid (Slack.UserComment uid) txt _ _ _) =
+    case words $ T.unpack txt of
```

Comment:
this might be too fancy, but thoughts on pulling this out into something that'll generate a command?

Thinking something like:

```haskell
data Command = Add Slack.UserId [String] | Remove Slack.UserId [String] | List Slack.ChannelId Slack.UserId | Stop Slack.UserId | Help [String]

parseCommand :: Text -> Command
parseCommand t =
    case words $ T.unpack t of
        "add":xs -> Add uid $ parsePrefs <$> xs
        "remove":xs -> Remove uid $ parsePrefs <$> xs
        -- etc
  where
    parsePrefs = rights . (parse parser "")
```

That way, parsing input can be decoupled from the action taken on the command and maybe make it more clear how the pieces interact.

Thoughts?